utils/validation/src/generators.rs

```diff
@@ -370,7 +372,7 @@ pub fn make_abi_test_fixtures() -> Result<TestFixtures, Error> {
         );
         stored_value.insert(
             "Withdraw".to_string(),
-            ABITestCase::from_inputs(vec![StoredValue::Withdraw(vec![
+            ABITestCase::from_inputs(vec![StoredValue::Unbonding(vec![
```

Comment:
Let's rename this test case from `"Withdraw"` to `"Unbonding"` (in the line above here) and add a test case for `StoredValue::Withdraw` please.

Reply:
Renamed, and added a test case